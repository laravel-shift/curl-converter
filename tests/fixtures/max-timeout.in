curl --max-timeout 5 https://example.com
