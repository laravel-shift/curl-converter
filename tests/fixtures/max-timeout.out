Http::timeout(5)
    ->get('https://example.com');
